mithril-test-lab/mithril-end-to-end/src/main.rs

```diff
@@ -56,6 +56,10 @@ pub struct Args {
     #[clap(long, default_value = "thales")]
     mithril_era: String,
 
+    /// Signed entity types parameters (discriminants names in an ordered comma separated list).
+    #[clap(long)]
+    signed_entity_types: Option<String>,
```

Comment:
It is possible to have Clap returning a Vec of Strings using [`value_delimiter`](https://docs.rs/clap/latest/clap/builder/struct.Arg.html#method.value_delimiter).

Reply:
Done :heavy_check_mark: